subsys/sd/sd.c

```diff
@@ -144,8 +159,8 @@ static int sd_init_io(struct sd_card *card)
 		LOG_ERR("Could not disable card power via SDHC");
 		return ret;
 	}
-	/* After reset or init, card voltage should be 3.3V */
-	card->card_voltage = SD_VOL_3_3_V;
+	/* After reset or init, card voltage should be max HC support */
```

Comment:
what is the reason for this change?

Reply:
eMMC host controller might support maximum 1.8V and hence change to accommodate setting voltage based on HC capabilities instead of hard coding to 3.3V.